MigrationGuide.md

```diff
@@ -0,0 +1,68 @@
+# Migration Guide
+
+This guide shows how to migrate to the new framework using CocoaPods, Carthage, or manually using the frameworks.
+
+
+### Using Cocoapods
+
+Change the entries in the pod file.
+
+| Old way              | New way                   |
+| :---                 | :---                      |
+| pod 'FacebookCore'   | pod 'FBSDKCoreKit/Swift'  |
+| pod 'FacebookLogin'  | pod 'FBSDKLoginKit/Swift' |
+| pod 'FacebookShare'  | pod 'FBSDKShareKit/Swift' |
+
+
+### Using Carthage
+
+We are choosing to not support Carthage for the Swift libraries until we drop support for Xcode 10.2. There are two main reasons for this: 
+
+a) After we drop support for Xcode 10.2, we can get rid of the concept of separate libraries and our existing tooling around Carthage will 'just work'
+
+b) Building on (a), we plan on supporting Swift Package Manager in the near future and as a small team, it makes more sense to put our time towards that effort.
+
+Carthage will continue to work against this codebase but you will not receive any updates. 
+
+If your goal is simply to link with prebuilt dynamic libraries, those can be found [here](https://github.com/facebook/facebook-ios-sdk/releases) in the file `SwiftDynamic.zip`. You will notice that the library names have changed, and the module names have changed. You can consume these in the same way you were previously consuming the libraries procured by Carthage.
+ 
+| Old Library / Module Name | New Library / Module Name |
+| :---                      | :---                      |
+| FacebookCore              | FBSDKCoreKit              |
+| FacebookLogin             | FBSDKLoginKit             |
+| FacebookShare             | FBSDKShareKit             |
+
+
+### Manual Way
+
+Under the [releases](https://github.com/facebook/facebook-objc-sdk/releases) tab on github you'll find a file named FBSDKCoreKit-Swift.zip (similar naming exists for Login and Share). If you unzip that you'll find FBSDKCoreKit.framework. This is the version of the framework that includes the Swift interface. You can confirm this by inspecting it in Finder. You will see a Modules folder that include FBSDKCoreKit.swiftmodule.
```

Comment:
Should you wrap "FBSDKCoreKit-Swift.zip" in code block similar to how `SwiftDynamic.zip` was?

That is "FBSDKCoreKit-Swift.zip" becomes `FBSDKCoreKit-Swift.zip`.

What about "FBSDKCoreKit.framework" and "FBSDKCoreKit.swiftmodule"?